VSTS/Tasks/SonarQubeScannerMsBuildBegin/task.json

```diff
@@ -0,0 +1,146 @@
+{
+    "id": "EAC0BC9E-D29E-48F8-920F-4833DA6B513A",
+    "name": "SonarQubeScannerMsBuildBegin",
+    "friendlyName": "[NEW] SonarQube Scanner for MSBuild - Begin Analysis",
+    "description": "Fetch the Quality Profile from SonarQube to configure the analysis",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=620063)",
+    "category": "Build",
+    "visibility": [
+        "Build"
+    ],
+    "author": "SonarSource",
+    "version": {
+        "Major": 0,
```

Comment:
I think the tasks are mature enough to have a 1.0.0 version. We're using sym versioning semantics, so 0.*.* means "work in progress", which these tasks are not.